tex/main/MarkovChain/closed_form_state_probs/main.tex

```diff
@@ -0,0 +1,662 @@
+\subsection{Graph Theoretical Approach}
+
+This section aims to investigate the connection between Markov chains and graph theory by exploring alternative ways of calculating the state probabilities and considering the problem from different perspectives.
+
+\subsubsection{Parameters}
+The inputs of the formula are the number of servers \(C\), the threshold \(T\), the system capacity \(N\) and the parking capacity \(M\). 
+Additional parameters of the model are the ambulance arrival rate, the others' arrival rate and the service rate, but for the purpose of this section these will remain unknown (\(\lambda^A, \lambda^o, \mu\)). 
+More specifically, the way these parameters are translated into the model are:
+
+\begin{itemize}
+    \item \textbf{Number of servers (\(C\)):} All service rates \(\mu\) in the Markov chain are multiplied by a coefficient equal to \(v\) for a state \((u,v)\) that stops increasing at \(v=C\). 
+    Thus, the coefficients of the service rate have a lower bound of \(0\) and an upper bound of \(C\).
+    \item \textbf{Threshold (\(T\)):} Determines the length of the left \textit{arm} of the model. 
+    In essence the threshold acts as a breakpoint between states where \(u=0\) and states where \(0 \leq u \leq M\). 
+    Increasing \(T\) results in having more set of states where \(u\) can only be \(0\).
+    \item \textbf{System capacity (\(N\)):} Is the upper bound of \(v\) for all states \((u,v)\).
+    \item \textbf{Parking capacity (\(M\)):} Is the upper bound of \(u\) for all states \((u,v)\) such that \(u \geq T\).
+\end{itemize}
+
+
+\subsubsection{Example figure of Markov Model}
+
+\begin{figure}[h]
+    \centering
+    \input{MarkovChain/closed_form_state_probs/example_model_1352/main.tex}
+    \caption{\(C=1, T=3, N=5, M=2\)}
+    \label{fig:Markov_1352_example_for_closed_form}
+\end{figure}
+
+In figure \ref{fig:Markov_1352_example_for_closed_form} an example of such Markov model is shown where \(C=1\) meaning the only coefficient in front of any \(\mu\) is going to be \(1\), \(T=3\) which means that the \textit{left arm} of the model has a length of \(3\), \(N=5\) that indicates that the right-most states \((u,v)\) are of the form \((u,5)\) and \(M=2\) that equivalently shows that the bottom states are of the form \((2,v)\).
+
+\subsubsection{Transforming the Markov chain model into graph theoretic model}
+
+An additional approach that one may consider to get the state probabilities is the graph theoretical approach for state probabilities.
+Thus, it can be assumed that a Markov chain model \(M\) can be translated as a weighted directed graph \(G_M\) where every edge has a weight that corresponds to the rate of the edges of the Markov chain. 
```

Comment:
```suggestion
Thus, it can be assumed that a Markov chain model \(M\) can be translated as a weighted 
directed graph \(G_M\) where every edge has a weight that corresponds to the 
rate between the states of a given edge.
```